core/templates/pages/email-dashboard-pages/email-dashboard-page.component.ts

```diff
@@ -20,6 +20,8 @@ require('base-components/base-content.directive.ts');
 
 require('services/user.service.ts');
 
+require('./email-dashboard-data.service');
```

Comment:
should we use an absolute import instead of a relative import

Reply:
@LanJosh Can you change this line to `require('pages/email-dashboard-pages/email-dashboard-data.service');`